airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -0,0 +1,348 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+from typing import Any, List, Mapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.models import ConfiguredAirbyteCatalog
+
+from .exceptions import TypeSalesforceException
+from .rate_limiting import default_backoff_handler
+
+STRING_TYPES = [
+    "byte",
```

Comment:
can we sort all these lists for better readability?

Reply:
What do you mean? Because each list is sorted.

Reply:
not all the lists are sorted e.g. https://github.com/airbytehq/airbyte/pull/5619/files#diff-e633b4da048d392e03b3632564f32322e76db05cea8141ca4b8f850618b16ef9R95